README.md

```diff
@@ -55,6 +55,8 @@ output {
       pravega_endpoint => "tcp://<host>:<port>"
       stream_name => "myStream"
       scope => "myScope"
+      #username => "admin"
+      #password => "1111_aaaa"
```

Comment:
Is there a reason to comment-out?   You can send a credential even when Pravega doesn't have auth enabled.

Reply:
No particular functional reason. It seems common to comment out non-required fields in logstash configurations.